Assets/Scripts/Tutorial-Power/TutorialGuide.cs

```diff
@@ -81,7 +81,7 @@ private void HandleDoorFirstClick()
     IEnumerator TriggerPowerOutage()
     {
         yield return new WaitForSeconds(1);
-        powerOutageEvent.TriggerEvent();
+        // powerOutageEvent.TriggerEvent();
```

Comment:
Still needed?

Reply:
had to change it since I made it so the power will already be off before the tutorial